atlasdb-dbkvs-hikari/src/main/java/com/palantir/nexus/db/pool/config/OracleConnectionConfig.java

```diff
@@ -112,6 +114,10 @@ public Properties getHikariProperties() {
 
         props.setProperty("oracle.jdbc.maxCachedBufferSize", "100000");
 
+        if (getDefaultRowPrefetch().isPresent()) {
+            props.setProperty("defaultRowPrefetch", getDefaultRowPrefetch().get().toString());
```

Comment:
In Postgres this is actually unlimited by default, so no test here should be able to cause an OOM that wouldn't have already happened without this being an option; the default in Oracle is 10, so we could potentially do tests for this internally with Oracle.

I also don't like the idea of setting an explicit default to make this the same in Postgres as it is in Oracle because I do not know if the perf implications are comparable between DB types.  It seems to make most sense to use the driver default when not set rather than always setting an explicit value (and in fact when documenting this we should recommend not setting this value unless you have an explicit reason).

This is also a good point that we may want to add other properties later. I will change this to allow specifying arbitrary properties instead.